### DISCUSSION TITLE
Remove box-shadow from <TableRow /> Component to reduce visual distraction

### DISCUSSION THREAD

**Discussion by AdventureBear** [radix-ui/themes#419] [Q&A]
I have a table with input and select fields. I'd like to get rid of the lines btween rows by removing the box-shadow.  Where / how is hte best way to do this?  I tried styling with tailwind using "box-shadow-none" as well as adding a style={{box-shadow: "none"}} to the TableRow. 

Is there another way to go about this?

**Comment by vladmoroz** [comment]
If the style overrides work for your case, that's fine I guess, but in general I'd recommend to build your own component for chromeless tables. 

The Reset component can be used to remove styles from the native table tags.

**Comment by thewebartisan7** [comment]
You can add inline style to table or table row like this:

`style={{ '--table-row-box-shadow': 'none' }}`
